pkg/cmd/attestation/verification/extensions.go

```diff
@@ -0,0 +1,27 @@
+package verification
+
+import (
+	"fmt"
+	"strings"
+)
+
+func VerifyCertExtensions(results []*AttestationProcessingResult, owner string, repo string) error {
+	for _, attestation := range results {
+		if owner != "" {
```

Comment:
the owner must always be set (compare with how the sourceRepositoryOwnerURI was set before), can't be allowed to be an empty string.

Reply:
remove the if condition